k8s/dialer.go

```diff
@@ -141,10 +147,10 @@ func (c *contextDialer) startDialerPod(ctx context.Context) (err error) {
 			Containers: []coreV1.Container{
 				{
 					Name:      c.podName,
-					Image:     socatImage,
+					Image:     img,
 					Stdin:     true,
 					StdinOnce: true,
-					Args:      []string{"-u", "-", "OPEN:/dev/null,append"},
```

Comment:
Pretty sure the new custom dialer fails with "flags not supportd" if the address has commas; so this is good this is now changed.

Reply:
it will only issue warning to stderr, not error

Reply:
also I removed the option in this PR :)

Reply:
that `,append` is totally unnecessary